LICENSE

```diff
@@ -577,4 +577,142 @@ be misrepresented as being the original software.
 3. This notice may not be removed or altered from any source
 distribution.
 
+H. stddef.h
```

Comment:
Please just call it musl and only include below what is between the horizontal lines, but remove lines.

Reply:
Done.

Reply:
Done